chapters/arrays.tex

```diff
@@ -1380,6 +1383,19 @@ \subsection{Slice Operation}\label{slice-operation}
 but it is still possible to perform some operations on them.
 \end{example}
 
+\begin{example}
+General selection
```

Comment:
There should be some punctuation at the end, and I think a colon is the common choice in a situation like this:
```suggestion
General selection:
```

Reply:
By the way, we should probably avoid introducing the new terminology _general selection_ in the example:
```suggestion
Member access slicing:
```